src/components/pages/Groups/Groups.tsx

```diff
@@ -40,14 +46,18 @@ const Groups = () => {
           onRefetch={() => void refetch()}
         />
       );
-    } else {
+    } else if (!tableData.length) {
       return (
-        <ModularTable
-          columns={COLUMN_DATA}
-          data={tableData}
-          emptyMsg={Label.NO_GROUPS}
+        <NoEntityCard
+          title="No groups"
+          message={Label.NO_GROUPS}
+          actionButton={
+            <Button appearance={ButtonAppearance.POSITIVE}>Create group</Button>
```

Comment:
I think it would be good to add a TODO here so we know to update this to display the panel when click.

Reply:
After the latest changes being merged in my PR, I added the actual functionality for opening the panel when pressing "Create group". Thanks for the reviews!